client/src/Login.js

```diff
@@ -0,0 +1,81 @@
+import "./Login.css";
```

Comment:
I can't see where login is connected to the running app, sorry if I missed it!

Reply:
No it's not connected, I'm not sure if I would continue with the log in bit.